setup.py

```diff
@@ -13,7 +13,8 @@
       package_dir = {'gaia_tools/': ''},
       packages=['gaia_tools','gaia_tools/load','gaia_tools/query',
                 'gaia_tools/xmatch','gaia_tools/select','gaia_tools/util'],
-      package_data={'gaia_tools/select':['2massc_hp5.txt'],
+      package_data={'gaia_tools/select': ['2massc_hp5.txt'],
+                    "gaia_tools/query/defaults": ["gaia_defaults.json"],
```

Comment:
Shouldn't this be
```
"gaia_tools/query": ["defaults/gaia_defaults.json"]
```
?

Reply:
done. And made the defaults directory platform agnostic.

Reply:
Thanks! Could you remove the ``defaults/__init__.py`` file again?